documentation/modules/auxiliary/admin/scada/advantech_webaccess_updatetemplate_sqli.md

```diff
@@ -0,0 +1,85 @@
+This module exploits a SQL injection vulnerability in Advantech WebAccess. The flaw can be found
```

Comment:
Maybe add a heading like "# Vulnerability Description" or something, but I really like that you went through all this detail on it!